as2/server/mail.md

```diff
@@ -0,0 +1,89 @@
+# Mail
+##### Messages that pertain to users' mail system.
+---
+### Delete Mail From Player
+Sent to clients when a postcard has been deleted from their penguin's mail.
+
+`mdp`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|newMailCount|Unsigned integer|No|
+
+##### Example
+`%xt%mdp%0%15%`
+
+### Send Mail
+Sent to clients after a send postcard request with either a successful result or an error.
```

Comment:
Maybe note the client's maximum postcard limit and say that the server is responsible for managing mailboxes accordingly.